sermant-plugins/sermant-flowcontrol/flowcontrol-common/src/main/java/io/sermant/flowcontrol/common/xds/retry/ResultRetryConditionType.java

```diff
@@ -34,7 +31,7 @@
  * @author zhp
  * @since 2024-11-29
  */
-public enum RetryConditionType {
+public enum ResultRetryConditionType {
     /**
```

Comment:
merge ResultRetryConditionType  with ExceptionRetryConditionType，and add a filed to distinguish retry on status code or exception

Reply:
done